specs/byron_minimal.cddl

```diff
@@ -0,0 +1,60 @@
+; Basic Cardano Types
+
+; blake2b224 = bytes .size 28
+; public_ed_25519_bip32 = bytes .size 64
+; script = bytes .size 32
+; public_ed_25519 = bytes .size 32
+
+blake2b224 = _CDDL_CODEGEN_RAW_BYTES_TYPE_
+public_ed_25519_bip32 = _CDDL_CODEGEN_RAW_BYTES_TYPE_
+byron_script = _CDDL_CODEGEN_RAW_BYTES_TYPE_
+public_ed_25519 = _CDDL_CODEGEN_RAW_BYTES_TYPE_
+
+; blake2b224 of sha256 of cbor(byron_addr_type, spending_data, addr_attributes)
+address_id = _CDDL_CODEGEN_RAW_BYTES_TYPE_ ; blake2b224 
+stakeholder_id = _CDDL_CODEGEN_RAW_BYTES_TYPE_; blake2b224
+crc32 = _CDDL_CODEGEN_EXTERN_TYPE_ ; uint .size 4 ; @newtype
+protocol_magic = _CDDL_CODEGEN_EXTERN_TYPE_ ; uint .size 4 ; @newtype
+hd_address_payload = bytes ; @newtype
+
+; Addresses
+
+; cddl had bootstrap as (1, uint) but byron/mod.rs had no uint field.
+stake_distribution = [
+  ; @name SingleKey
+  tag: 0, stakeholder_id //
+  ; @name BootstrapEra
+  bootstrap_era: 1
+]
```

Comment:
I had to put the field name as `bootstrap_era` too since cddl-codegen's dcSpark/cddl-codegen#153 ignores `@name` for single-field group choices. i.e. `tag: 1` as the field would make the entire variant come out as `StakeDistribution::Tag` instead of just the fixed field during serialization.